horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/TestService.java

```diff
@@ -228,18 +229,18 @@ List<ExportedLabelValues> listLabelValues(@PathParam("id") int testId,
 
    @GET
    @Path("{id}/export")
-   @APIResponseSchema(value = String.class,
+   @APIResponseSchema(value = TestExport.class,
            responseDescription = "A Test defintion formatted as json",
            responseCode = "200")
-   String export(@PathParam("id") int testId);
+   TestExport export(@PathParam("id") int testId);
```

Comment:
I think we should still keep this generic, but use OjectNode instead of String, so that we are not tied to a single TestExport structure and can possibly support multiple versions of TestExport.
I think we need to consider backwards compatibility between Horreum Versions

Reply:
Personally I like typed APIs. Using ObjectNode would be an "articial" abstraction as it would be deserialized to TestExport anyway. If we add a field to TestExport in a later version an older version would still be accepted by Horreum, but it would not populate the new field.

Reply:
The TestExport class can be mutated but adding, removing fields, changing field types and changing relationships in Entity graph that TestExport represents. deserialization breaks when you change types and relationships, and we would have not control over it

Reply:
The api can still be typed, e.g.;

```
    @NotNull
    @JsonProperty( required = true )
    @Schema(type = SchemaType.OBJECT,
            oneOf = {
                TestExport_v1.class,
                    TestExport_v2.class
    })
    public ObjectNode export(@PathParam("id") int testId);
 ```

Reply:
See: https://horreum.hyperfoil.io/openapi/#tag/Config/operation/newDatastore as an example